### PR TITLE
Contributor spotlight for Rajiv Singh

### PR DESCRIPTION
**Pull request by alyssat**
Create contributor spotlight story for Rajiv Singh

### CODE REVIEW

**Review by gounthar — APPROVED** [review]
Looks good to me, thanks Alyssa! 👍

**Comment by krisstern** [comment]
I think Rajiv's spotlight should be turned on by updating `:page-featured: false` to `:page-featured: true`, while Harsh's spotlight should be turned off.

**Comment by gounthar** [comment]
> I think Rajiv's spotlight should be turned on by updating `:page-featured: false` to `:page-featured: true`, while Harsh's spotlight should be turned off.

We should do that next week when we're ready to publish, right?

**Review by lemeurherve — DISMISSED** [review]

**Comment by krisstern** [comment]
> > I think Rajiv's spotlight should be turned on by updating `:page-featured: false` to `:page-featured: true`, while Harsh's spotlight should be turned off.
> 
> We should do that next week when we're ready to publish, right?

No, we don't have to wait because the changes will not be shown until the PR is merged.

File: src/pages/contributors/rajiv-singh.adoc [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+:page-linkedin: iamrajivranjansingh
+:page-twitter: therajiv
+:page-github: iamrajiv
+:page-email: 
+:page-image: avatar/rajiv-singh.png
+:page-pronouns: He/Him/His
+:page-location: Bengaluru, India
+:page-firstcommit: 2023
+:page-datepublished: 2024-07-02
+:page-featured: false
```
Comment (gounthar) [inline src/pages/contributors/rajiv-singh.adoc:12]:
```suggestion
:page-featured: true
```

**Comment by alyssat** [comment]
> I think Rajiv's spotlight should be turned on by updating `:page-featured: false` to `:page-featured: true`, while Harsh's spotlight should be turned off.

I've changed Harsh's spotlight false

**Comment by krisstern** [comment]
> > I think Rajiv's spotlight should be turned on by updating `:page-featured: false` to `:page-featured: true`, while Harsh's spotlight should be turned off.
> 
> I've changed Harsh's spotlight false

Hi @alyssat Could we do that in this PR instead of a new one #152?

File: src/pages/contributors/rajiv-singh.adoc [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+:page-twitter: therajiv
+:page-github: iamrajiv
+:page-email: 
+:page-image: avatar/rajiv-singh.png
+:page-pronouns: He/Him/His
+:page-location: Bengaluru, India
+:page-firstcommit: 2023
+:page-datepublished: 2024-07-02
+:page-featured: true
+:page-intro: Rajiv works as a software engineer at A.P. Moller - Maersk based in India. In 2022 he graduated from JSS Academy of Technical Education with a Bachelor in Information Science & Engineering. Fascinated by the extensive impact computers have on solving real world problems, he writes software for a living and has primarily worked on backend systems and infrastructure. His focus is on the fields of algorithms and distributed systems.
```
Comment (krisstern) [inline src/pages/contributors/rajiv-singh.adoc:13]:
```suggestion
:page-intro: Rajiv works as a software engineer at A.P. Moller - Maersk based in India. In 2022 he graduated from JSS Academy of Technical Education with a Bachelor in Information Science and Engineering. Fascinated by the extensive impact computers have on solving real world problems, he writes software for a living and has primarily worked on backend systems and infrastructure. His focus is on the fields of algorithms and distributed systems.
```

**Review by krisstern — CHANGES_REQUESTED** [review]

**Review by krisstern — APPROVED** [review]
LGTM

**Comment by krisstern** [comment]
@lemeurherve Please review again and approve of the changes made

**Review by lemeurherve — APPROVED** [review]

**Pull request merged**